django/contrib/gis/db/models/functions.py

```diff
@@ -274,9 +276,20 @@ class Envelope(GeomOutputGeoFunc):
     arity = 1
 
 
+class ForcePolygonCW(GeomOutputGeoFunc):
+    arity = 1
+
+
 class ForceRHR(GeomOutputGeoFunc):
     arity = 1
 
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            'ForceRHR() is deprecated in favor of ForcePolygonCW.',
+            RemovedInDjango30Warning, stacklevel=2,
+        )
+        super().__init__(*args, **kwargs)
```

Comment:
Is your concern that `ForcePolygonCW` won't be discoverable for users of older versions of PostGIS? I guess the main disadvantage of a delayed deprecation is continued use of a deprecated alias which would could result in more places to change when the deprecation happens. Could be mitigated with documentation though.

Reply:
I thought the change in `postgis/operations.py` is meant to alias `ForcePolygonCW` to `ST_ForceRHR` on older PostGIS versions to avoid the problem you described.